tests/test_pipermail.py

```diff
@@ -17,6 +18,10 @@
 #
 # Authors:
 #     Santiago Dueñas <sduenas@bitergia.com>
+#     Stephan Barth <stephan.barth@gmail.com>
+#     Valerio Cosentino <valcos@bitergia.com>
+#     Miguel Ángel Fernández <mafesan@bitergia.com>
+#     Harshal Mittal <harshalmittal4@gmail.com>
```

Comment:
I checked the test_pipermail.py commit history and I see zhquan as one of the author, but the list doesn't have his name.
https://github.com/chaoss/grimoirelab-perceval/commit/fca976a0a0e5dbfc6b895da986a502030740f22c

Also, for reference [github api commits - test_pipermail.py](https://api.github.com/repos/chaoss/grimoirelab-perceval/commits?path=tests/test_pipermail.py)

Did I assume anything wrong over here?

Reply:
I have to check the tests for pipermail. Tomorrow I'll have a look and let you know. Thanks for raising this question!

Reply:
sorry for the late reply! ok, I understand that he should be added.

Reply:
Thanks for the reply.

I'll update the PR and let you know soon. :slightly_smiling_face: